### ISSUE TITLE
Continuously increasing memory usage? 

### ISSUE THREAD

**Issue by jswag245** [lllyasviel/stable-diffusion-webui-forge#1955]
I'm having a strange issue. I've noticed that I can generate 1-3 times using the default GPU weight and the same parameters/model for each one without any problems. However, I eventually run into either the low VRAM error or OOM Cuda error. Switching models seems to make the issue worse, meaning I can generate fewer times before I run into either error. 

I understand that normally, I would have to lower my GPU weight. However, it doesn't take long before any amount of lowering does nothing. The solution is temporarily fixed by restarting my computer, but the issue persists. I wonder if this is the memory leak issue others have mentioned before? 

I would like to add I am not using xformers, and haven't added any arguments.

**Comment by elen07zz** [comment]
This always happens to me when I use flux, my solution was simply to change to comfyui, and I don't like using comfyui, but at least there are no memory problems.

**Comment by wikkidwookie24** [comment]
I was getting an issue like this for some reason my issues stopped after I turned off indexing and super fetch in windows. Don't know if that is what ultimately fixed my memory issues I just know after I turned those off I wasn't getting a ran out of cuda memory after 3 gens.

**Comment by byteconcepts** [comment]
Same here,
actual GPU memory management is imho broken.

Nvidia 4090 24G - Ubuntu 22.04 Linux

**Comment by andy8992** [comment]
Been getting OOM for the first time ever recently, but may be just me.

**Comment by EianAtDawn** [comment]
> Been getting OOM for the first time ever recently, but may be just me.

me too.. also seems to trigger the "153 from source nvlddmkm" error in the event viewer..
maybe something to do with recent drivers?? or something else??

**Comment by jswag245** [comment]
I am closing this issue due to not having it occur recently. Also, I am running a laptop with two batteries, I've noticed that running SD will drain the battery that supplies my GPU much faster than the other, which makes sense of course. This will eventually cause the battery to drop to 0%, cutting off the GPU, giving me the "press any key to continue." At first, I thought this was due to weights or page files, but nope, I'm just using a laptop.
